samples/tfm/tfm_secure_peripheral/prj.conf

```diff
@@ -10,3 +10,4 @@ CONFIG_TFM_IPC=y
 CONFIG_NRF_TIMER1_SECURE=y
 CONFIG_NRF_SPIM3_SECURE=y
 CONFIG_NRF_GPIOTE0_SECURE=y
+CONFIG_TFM_ISOLATION_LEVEL=1
```

Comment:
This could also be dropped since we have default to 1 for all TF-M enabled devices. I don't have a very strong opinion on this so I don't mind if you want to keep it though.